### ISSUE TITLE
[US - UXD] As MILES HARDIN, I want to see my idea’s status in a tag so that I can know where my idea is in the process to be approved by Dev Launchers.

### ISSUE THREAD

**Issue by JulieMass** [dev-launchers/dev-launchers-platform#780]
### Considerations:
We need to include a tag for each idea in the dashboard page so Miles can know the status of each one. For example: is an idea in workshopping, development, archived, closed, etc. phase? If it is in "workshopping phase" then the label should show "workshopping" and so on. 

### Acceptance criteria:
- Visual designs for the dashboard are updated with the status flags.
- Mockups are web responsive and include mobile, tablet and desktop breakpoints.

----------------------------------------------------
### Sub-tasks:
- [ ] UX design: Hi-fi mockups updated for the dashboard page

----------------------------------------------------
![image.png](https://images.zenhubusercontent.com/63400f3d1e79614e45d9ebd9/69d51125-65e9-4b0d-87e5-57861305513c)

[Idea's statuses](https://www.figma.com/file/uI7sKOvgtHhQYROxGgX2wq/%5BDL---IdeaSpace%5D-Story-mapping?node-id=0%3A1&t=BNigCJQfnvRLQhiz-1) need to be reviewed and defined:
<img width="927" alt="image" src="https://user-images.githubusercontent.com/104226366/209822726-8ab225e1-bf84-4ee5-a323-0c0d025e8e5e.png">

**Comment by JulieMass** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/IdeaSpace-Team-6350185d9843157427cc900b/issues/dev-launchers/dev-launchers-platform/780?planning-poker) with Zenhub @alicefate @dbradham @jordanmcousins @JulieMass @liwenlin664477 @Nouri-Anouar @pyxld-kris @RETprojects

**Comment by JulieMass** [comment]
Hey @Nouri-Anouar would it be OK if you share here a screenshot of how each idea tag state should look like so in this way we can use it to validate if the development is aligned later? because I was able to find the dashboard page with some of the tags like "approved"(green) "workshopping" (orange) and "submitted" (blue) but I couldn't find how the "archived" or "applying" tag should look like. 

By the way, I like that you selected green for the "approved" and the blue for the "submitted". Makes much more sense than the colors I had in the workflow map 😁 👌. I updated the idea's status map to follow your colors ☺️
